packages/viewer/tests/cypress/tests-e2e/3d/layers.cy.ts

```diff
@@ -1,17 +1,15 @@
 import { WEBMERCATOR, LV95 } from '@swissgeo/coordinates'
-
+// @ts-expect-error Editable Feature class not a typescript file for now
```

Comment:
wouldn't it fix the error if you added `.js` at the end of the import?

or we could also add `allowJS: true` to our `tsconfig.json` used for Cypress